plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -78,6 +78,12 @@
     choices: ["maintainer", "owner"]
     type: str
     version_added: 3.7.0
+  require_two_factor_authentication:
+    description:
+      - Require all users in this group to setup two-factor authentication.
+    type: bool
+    default: false
```

Comment:
```suggestion
```
State options shouldn't have a default. Otherwise you're breaking backwards compatibility by suddenly updating groups that had this value set to `true` somehow else. The value should only be modified if the user explicitly specified a value here.

Reply:
Fixed with latest commit.